doc/cascadia/profiles.schema.json

```diff
@@ -10,7 +10,7 @@
       "format": "color"
     },
     "Coordinates": {
-      "pattern": "^(-?\\d+)?,?(-?\\d+)?$",
+      "pattern": "^(?!-?\\d+-\\d+$)(-?\\d+)?,?(-?\\d+)?$",
```

Comment:
Hold up, don't do that. Lookarounds are expensive.

The correct way to fix this is -- assuming you want to support `5` `5,5` and `,5` -- to include the `,` inside one of the optional groups.

Use this pattern.

`^(-?\\d+)?(,?\\s?-?\\d+)?$`

Including the `,` in the second optional group (also: `\s` matches a single space, and is valuable to put in places users might put spaces) means you don't need to have an expensive negative lookaround in the regex that says "plx make sure it doesn't contain 5-5"

Reply:
Using `?` for `\s` makes it so that they can only say `5, 5` or `5,5` -- they can't say `5,            5`.

Unfortunately, they _can_ say `5,\n5` :grin:

Reply:
We also have to allow for `,5` 😟

Reply:
Yes, that is part of the thing I said ;P

Reply:
Oh oh oh I see, my bad :smile:

Reply:
Now, this regex will match `""`... which we probably don't want. Because both parts are optional.

To require _at least_ one digit in each component... sec

Reply:
i think empty is fine (it means 0,0 if i recall properly), so the last pattern stands

Reply:
Wait, the pattern you have doesn't allow for `5,`

Also, having `""` will just use the default settings, so that's probably fine?

Reply:
ah, `5,` is legal.

`^(-?\\d+)?(,\\s?(-?\\d+)?)?$`

Reply:
"if there's a second digit group it must be preceded by `,\s+` but there doesn't need to be digits after the ,"

Reply:
![image](https://user-images.githubusercontent.com/14316954/73297119-d49f7380-41bf-11ea-82df-8bdf565773a4.png)

Reply:
Ah okay yeah that works :) I'll add it in! Thanks 😄